CMakeLists.txt

```diff
@@ -854,24 +854,54 @@ set(wrapper_test_scripts
 	test-fake-ssh test-line-buffer test-svn-fe test-tool)
 
 
-foreach(script ${wrapper_scripts})
-	file(STRINGS ${CMAKE_SOURCE_DIR}/wrap-for-bin.sh content NEWLINE_CONSUME)
-	string(REPLACE "@@BUILD_DIR@@" "${CMAKE_BINARY_DIR}" content "${content}")
-	string(REPLACE "@@PROG@@" "${script}${EXE_EXTENSION}" content "${content}")
-	file(WRITE ${CMAKE_BINARY_DIR}/bin-wrappers/${script} ${content})
-endforeach()
+if(NOT DEFINED CI_BUILD)
 
-foreach(script ${wrapper_test_scripts})
+	foreach(script ${wrapper_scripts})
+		file(STRINGS ${CMAKE_SOURCE_DIR}/wrap-for-bin.sh content NEWLINE_CONSUME)
+		string(REPLACE "@@BUILD_DIR@@" "${CMAKE_BINARY_DIR}" content "${content}")
+		string(REPLACE "@@PROG@@" "${script}${EXE_EXTENSION}" content "${content}")
+		file(WRITE ${CMAKE_BINARY_DIR}/bin-wrappers/${script} ${content})
+	endforeach()
+	
+	foreach(script ${wrapper_test_scripts})
+		file(STRINGS ${CMAKE_SOURCE_DIR}/wrap-for-bin.sh content NEWLINE_CONSUME)
+		string(REPLACE "@@BUILD_DIR@@" "${CMAKE_BINARY_DIR}" content "${content}")
```

Comment:
It's the same here as in the `wrapper_scripts` part above. Maybe we should define what we want to replace `@@BUILD_DIR@@` with, above, platform-dependent. And then not have two almost identical blocks transmogrifying the `bin-wrapper/` scripts?

Reply:
The replacement for @@BUILD_DIR@@ is fine. 
The reason for the 2 almost identical blocks is because of the expected directory for the test tools is different from the git executables